includes/gateways/stripe/includes/admin/admin-actions.php

```diff
@@ -388,3 +430,102 @@ function give_stripe_show_currency_notice() {
 }
 
 add_action( 'admin_notices', 'give_stripe_show_currency_notice' );
+
+/**
+ * Disconnect Stripe Account.
+ *
+ * @since 2.7.0
+ *
+ * @return void
+ */
+function give_stripe_disconnect_connect_stripe_account() {
+
+	$get_data = give_clean( $_GET );
+
+	if ( is_admin() && isset( $get_data['stripe_disconnected'] ) ) {
+		$account_name = ! empty( $get_data['account_name'] ) ? $get_data['account_name'] : false;
+
+		// Disconnect Stripe Account.
+		give_stripe_disconnect_account( $account_name );
```

Comment:
This value should be sanitized.

Reply:
@brad already sanitized at beginning of this function.

Reply:
@bradp The complete `$get_data` array is sanitized already so no need to sanitize it separately.